rules/base.js

```diff
@@ -53,23 +56,125 @@ module.exports = {
       },
     ],
 
-    // TODO: come back to it when we start to extend prettier recommended
-    // Need to handle a case with Joi Models
-    'newline-per-chained-call': 'off',
-
     // Ref: https://eslint.org/docs/rules/no-prototype-builtins
+    //
     // Eslint reasoning: https://github.com/eslint/eslint/issues/7071
+    //
     // Airbnb: 'no-prototype-builtins': 'error',
-    // Example (if enable): obj.hasOwnProperty(key) => (should be) Object.prototype.hasOwnProperty.call(obj, key)
     //
-    // We prefer to get Error and get aware of the wrong usage. It is much better then silently hadle dangerous code.
+    // Example (if enable):
+    // obj.hasOwnProperty(key) => (should be) Object.prototype.hasOwnProperty.call(obj, key)
+    //
+    // We prefer to get Error and get aware of the wrong usage. It is much better then silently
+    // hadle dangerous code.
     'no-prototype-builtins': 'off',
```

Comment:
Should we enable it?

Reply:
We decided long time that we want to keep it disabled.

Reply:
settled